.devcontainer/Dockerfile

```diff
@@ -1,23 +0,0 @@
-﻿# See here for image contents: https://github.com/microsoft/vscode-dev-containers/tree/v0.154.0/containers/dotnet/.devcontainer/base.Dockerfile
```

Comment:
Everything in `.devcontainer` was leftover from the old blazorwasm debugging extension